packages/rocketchat-contacts/server/startup.js

```diff
@@ -0,0 +1,12 @@
+RocketChat.settings.addGroup('Contacts', function() {
+	this.add('Contacts_Phone_Custom_Field_Name', '', {
```

Comment:
@kb0304 Can you explain how you use this, "Contacts_Phone_Custom_Field_Name"?  Do we edit this to be the path in the db to the telephoneNumber?  i.e. "services.viasatsso.telephonNumber", something like that?  Maybe I should know this.......  but thanks.